app/controllers/application_controller.rb

```diff
@@ -36,18 +40,44 @@ def rescue_steps(message)
   def record_activity
     activity_param = {
         activity_type: "#{params[:controller]}\##{params[:action]}",
-        user_id: current_user.nil? ? nil : current_user[:id],
+        user_id: current_user.nil? ? nil : current_user['id'].to_i,
```

Comment:
When we push the new updates to production, will the old data be a problem?

Reply:
Possibly. I'll revert this change back.